sw/airborne/modules/computer_vision/cv.c

```diff
@@ -26,28 +26,52 @@
  */
 
 #include "cv.h"
+#include <stdlib.h> // for malloc
 
-#define MAX_CV_FUNC 10
+void cv_add_to_device(struct video_config_t *device, cvFunction func)
+{
+  // Initialise the device that we want our function to use
+  video_thread_initialize_device(device);
```

Comment:
This is currently depending on video_thread.c, which conflicts with the intended structure (video thread depending on cv). It gives a warning during compilation time. Any suggestions on how to refactor this?